Helper/Order.php

```diff
@@ -549,17 +549,13 @@ private function deleteRedundantQuotes($quote) {
      */
     public function saveUpdateOrder($reference, $frontend = true, $bolt_trace_id = null)
     {
-
         $transaction = $this->fetchTransactionInfo($reference);
         $incrementId = $transaction->order->cart->display_id;
 
-        // load the quote from reserved order id
-        // $quote = $this->loadQuote($incrementId);
-
         // Load quote from entity id
         $quoteId = $transaction->order->cart->order_reference;
-        $quote = $this->quoteFactory->create()->load($quoteId);
-        //$quote = $this->quoteRepository->get($quoteId);
+        //$quote = $this->quoteFactory->create()->load($quoteId);
+        $quote = $this->quoteRepository->get($quoteId);
```

Comment:
what is the reason for this change?

Reply:
Model::load is deprecated (but still heavily used). The repository has already been available. Removed the factory completely.